### ISSUE TITLE
Are PATCH .sdb's no longer work in Windows 10?

### ISSUE THREAD

**Issue by rustyx** [evil-e/sdb-explorer#2]
No matter what I try, my PATCH SDBs are being ignored in Windows 10. All other SDB's (InjectDll, etc.) still work fine.
Is this a known issue or am I doing something wrong?

**Comment by evil-e** [comment]
I do not believe they work in Windows 10.  See this blog: http://subt0x10.blogspot.com/ from Casey Smith

**Comment by rustyx** [comment]
Interestingly, the `sdbinst.exe` utility still has the `-p` flag to install PATCH SDBs, suggesting that the support is still there, so I'm wondering if maybe just the format has changed or it needs a special tag, a signature (finally?) or something along those lines.

**Comment by evil-e** [comment]
I've compared apphelp.dll on Windows 7 versus Windows 10.  The Windows 10 version does not have the function SeiApplyPatch.  This is what is responsible for performing the PATCH.  It appears that Microsoft has removed this feature.

**Comment by uudiin** [comment]
The sdb maked by sdb-explorer is no longer work in win10, but I found the EMET's SDB work fine in win10, So I made a same sdb file as EMET's sdb, it doesn't work also. It's confused.
I beleve that there is a way to make it work in win10.
